admin_app/src/components/ContentCard.tsx

```diff
@@ -127,10 +137,11 @@ const ContentCard = ({
             size="small"
```

Comment:
Looked nicer with "medium" with the new icon. But i'll let you decide!

Reply:
yes, medium looks better